xclim/indices/_threshold.py

```diff
@@ -30,6 +30,8 @@
     threshold_count,
 )
 
+from .helpers import resample_map
```

Comment:
I think we should move towards absolute imports.

```suggestion
from xclim.indices.helpers import resample_map
```